### PR TITLE
Saas 13.5 state planned whe

### PR DESCRIPTION
**Pull request by Whenrow**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/models/stock_picking.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def _get_mo_count(self):
             return
         domains = {
             'count_mo_waiting': [('reservation_state', '=', 'waiting')],
-            'count_mo_todo': ['|', ('state', 'in', ('confirmed', 'draft', 'planned', 'progress'))],
+            'count_mo_todo': ['|', ('state', 'in', ('confirmed', 'draft', 'progress')), ('is_planned', '=', True)],
```
Comment (ryv-odoo) [inline addons/mrp/models/stock_picking.py:34]: A planned MO is also confirmed, it is useless to add this condition

File: addons/mrp/views/mrp_production_views.xml [hunk 2]
```diff
@@ -468,13 +467,13 @@
                     <field name="move_raw_ids" string="Component" filter_domain="[('move_raw_ids.product_id', 'ilike', self)]"/>
                     <field name="name" string="Work Center" filter_domain="[('bom_id.operation_ids.workcenter_id', 'ilike', self)]"/>
                     <field name="origin"/>
-                    <filter string="To Do" name="todo" domain="[('state', 'in', ('draft', 'confirmed', 'planned','progress', 'to_close'))]"
+                    <filter string="To Do" name="todo" domain="['|', ('state', 'in', ('draft', 'confirmed', 'progress', 'to_close')), ('is_planned', '=', True)]"
```
Comment (ryv-odoo) [inline addons/mrp/views/mrp_production_views.xml:470]: > A planned MO is also confirmed, it is useless to add this condition.

The same ;)

File: addons/mrp/views/mrp_production_views.xml [hunk 3]
```diff
@@ -331,7 +330,7 @@
                                     <field name="has_move_lines" invisible="1"/>
                                     <field name="location_dest_id" domain="[('id', 'child_of', parent.location_dest_id)]" invisible="1"/>
                                     <field name="state" invisible="1" force_save="1"/>
-                                    <field name="product_uom_qty" string="To Produce" attrs="{'readonly': ['&amp;', ('parent.state', '!=', 'draft'), '|', ('parent.state', 'not in', ('confirmed', 'planned', 'progress', 'to_close')), ('parent.is_locked', '=', True)]}"/>
+                                    <field name="product_uom_qty" string="To Produce" attrs="{'readonly': ['&amp;', ('parent.state', '!=', 'draft'), '|', '&amp;', ('parent.state', 'not in', ('confirmed', 'progress', 'to_close')), ('parent.is_planned', '!=', True), ('parent.is_locked', '=', True)]}"/>
```
Comment (ryv-odoo) [inline addons/mrp/views/mrp_production_views.xml:333]: > A planned MO is also confirmed, it is useless to add.

The same

File: addons/mrp/views/mrp_production_views.xml [hunk 4]
```diff
@@ -262,7 +261,7 @@
                                     <field name="location_dest_id" domain="[('id', 'child_of', parent.location_dest_id)]" invisible="1"/>
                                     <field name="state" invisible="1" force_save="1"/>
                                     <field name="should_consume_qty" invisible="1"/>
-                                    <field name="product_uom_qty" widget="mrp_should_consume" string="To Consume" attrs="{'readonly': ['&amp;', ('parent.state', '!=', 'draft'), '|', ('parent.state', 'not in', ('confirmed', 'planned', 'progress', 'to_close')), ('parent.is_locked', '=', True)]}" width="1"/>
+                                    <field name="product_uom_qty" widget="mrp_should_consume" string="To Consume" attrs="{'readonly': ['&amp;', ('parent.state', '!=', 'draft'), '|', '&amp;', ('parent.state', 'not in', ('confirmed', 'progress', 'to_close')), ('parent.is_planned', '!=', True), ('parent.is_locked', '=', True)]}" width="1"/>
```
Comment (ryv-odoo) [inline addons/mrp/views/mrp_production_views.xml:264]: > A planned MO is also confirmed, it is useless to add this condition.

The same

File: addons/mrp/models/mrp_production.py [hunk 5]
```diff
@@ -314,14 +314,28 @@ def _set_date_deadline(self):
         for production in self:
             production.move_finished_ids.date_deadline = production.date_deadline
 
+    @api.depends("workorder_ids")
```
Comment (ryv-odoo) [inline addons/mrp/models/mrp_production.py:317]: `"workorder_ids", "workorder_ids.date_planned_start", "workorder_ids.date_planned_finished"`

**Review by ryv-odoo — APPROVED** [review]
Nice !

**Comment by Whenrow** [comment]
robodoo r+ rebase-ff

**Comment by Whenrow** [comment]
@fw-bot ignore

**Pull request closed without merging**